README.md

```diff
@@ -128,87 +130,79 @@ Number matching is weak - the number has to appear
 exactly the same in the pattern and the event. I.e.,
 Quamina doesn't know that 35, 35.000, and 3.5e1 are the
 same number. There's a fix for this in the code which 
-is commented out because it causes a 
-significant performance penalty.
+is not yet activated because it causes a 
+significant performance penalty, so the API needs to
+be enhanced to only ask for it when you need it.
 
 ## Flattening and Matching
 
 The first step in finding matches for an Event is 
 “flattening” it, which is to say turning it 
 into a list of pathname/value pairs called Fields. Quamina 
-defines a `Flattener` interface type and provides a 
-JSON-specific implementation in the `FJ` type.
+defines a `Flattener` interface type and has a built-in
+`Flattener` for JSON.
 
 `Flattener` implementations in general will have
 internal state and thus not be thread-safe.
```

Comment:
nit: "thus not be thread-safe"? Since this is an interface, IMHO we cannot tell whether an implementation is/must be thread-safe.

Reply:
I said "in general will have internal state" - it's possible you could have a thread-safe flattener but I don't think the language rules it out.

Reply:
Got it. But in "thus not be thread-safe" there's still something missing, e.g. "thus are not thread-safe"?